apps/pipelines/models.py

```diff
@@ -395,6 +423,28 @@ def archive(self):
                 assistant = OpenAiAssistant.objects.get(id=assistant_id)
                 assistant.archive()
 
+    @property
+    def version(self) -> Version:
+        from apps.experiments.models import VersionFieldDisplayFormatters
+
+        node_name = self.params.get("name", self.flow_id)
```

Comment:
The default name is the node ID which is quite ugly. A 'to_display' function for this would be nice to apply similar logic to the UI:
```python
node_type = node_cls.model_config['json_schema_extra'].label
if not name or name == node_id:
    return node_type
else:
    return f"{name} ({node_type})"
```

Reply:
https://github.com/dimagi/open-chat-studio/pull/1085/commits/cf01f5ff9b10048676c439f6735dee55b97335e1